sound/soc/sof/ipc4.c

```diff
@@ -542,6 +542,40 @@ static int sof_ipc4_init_msg_memory(struct snd_sof_dev *sdev)
 	return 0;
 }
 
+u32 sof_ipc4_find_debug_slot_offset_by_type(struct snd_sof_dev *sdev, void *cached_addr,
+					    u32 slot_type)
+{
+	u32 slot_desc_type_offset;
+	u32 start_offset;
+	u32 type;
+	int i;
+
+	if (pm_runtime_active(sdev->dev))
+		start_offset = sdev->debug_box.offset;
```

Comment:
I'm not sure about this pm_runtime_active() stuff. This may be ok, but certainly pops out from the code. So not blocking on this, but logging a concern.